doc/release-notes-26471.md

```diff
@@ -0,0 +1,14 @@
+Updated settings
+----------------
+
+- Previously running with the `-blocksonly` option effectively disabled the
```

Comment:
```suggestion
- Previously running with the `-blocksonly` option all but disabled the
```

Reply:
Personally I feel "effectively disabled" is cleared here, so I've left it as-is for now (but happy to be persuaded)

Reply:
The definition "actually but not officially or explicitly" captures "effectively" quite well for me, and in that sense I think is wrong here. But won't comment further, not that important.

Reply:
Removed in new version